app/src/main/java/sw19/moring03/paint/tools/TriangleTool.java

```diff
@@ -0,0 +1,60 @@
+package sw19.moring03.paint.tools;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Path;
+
+import java.util.ArrayList;
+
+import sw19.moring03.paint.utils.PointF;
+
+public class TriangleTool extends Tools {
```

Comment:
i can change that if you give me an explicit way how you want it to look like. Currently using this calculation since it made the most sense to me since you can create triangles quite freely.